spec/requests/cors_spec.rb

```diff
@@ -0,0 +1,157 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe 'CORS requests' do
+  expose_headers = (MediaPoll::HEADERS_EXPOSED + ['X-Archived-At', 'X-Error-Type']).join(', ')
+  allow_methods = ['GET', 'POST', 'PUT', 'PATCH', 'HEAD', 'DELETE', 'OPTIONS'].join(', ')
+
+  # have to specify content type header, otherwise it gets set to application/x-www-form-urlencoded
+
+  context 'CORS browser example' do
+    headers = {
+      'Host' => 'localhost:8080',
+      'Connection' => 'keep-alive',
+      'Cache-Control' => 'max-age=0',
+      'Origin' => 'http://localhost:8080',
+      'User-Agent' => 'Mozilla/5.0 (Windows NT 6.3; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/39.0.2171.95 Safari/537.36',
+      'Access-Control-Request-Headers' => 'accept, content-type',
+      'Accept' => '*/*',
+      'Referer' => 'http://localhost:8080/listen/234234?start=30&end=60',
+      'Accept-Encoding' => 'gzip, deflate, sdch',
+      'Accept-Language' => 'en-US,en;q=0.8',
+      'Content-Type' => 'text/plain'
+    }
+
+    it '1, has the expected CORS response' do
```

Comment:
Test names are a little confusing, what is 1 and 2?

Reply:
this file was renamed, and not part of the change set.

I didn't write these tests but I believe the numbers are different examples of requests